client/platform/desktop/frontend/components/ViewerLoader.vue

```diff
@@ -29,6 +32,13 @@ export default defineComponent({
 
 <template>
   <Viewer :id="id">
+    <template #extension-right>
+      <Export
+        v-if="datasets[id]"
+        :id="id"
+        :small="true"
+      />
+    </template>
```

Comment:
![Screenshot from 2021-02-01 13-16-21](https://user-images.githubusercontent.com/4214172/106500335-b08c0780-648f-11eb-9864-ed38a7d38088.png)

Arbitrary choice.  On small displays, something will always suffer.  Can change it if you want.

The above looked nicer to me.

Reply:
TBH I think we can get rid of "DIVE" on the left hand side for desktop.  It serves no real purpose.

Reply:
![image](https://user-images.githubusercontent.com/61746913/106500524-f052ef00-648f-11eb-9ca0-850b85820761.png)
I just checked because I forgot, we do have the download button next to pipelines in the web version.  I truly don't feel strongly about it one way or another so whatever you decide is fine with me.